toolkit/resources/imageconfigs/postinstallscripts/iso_initrd/cleanup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Is it worth cleaning up the rpm cache here as well? We do that for containers.

Reply:
Not seeing cache under /var in initrd image.